### ISSUE TITLE
DataSet

### ISSUE THREAD

**Issue by blankmeadow** [Arka95/Human-Emotion-Analysis-using-EEG-from-DEAP-dataset#1]
Hi, can you kindly show me the download link for DEAP dataset? I cannot get any reply after sending request emails in the official website. And I would appreciate your help cause I really need the dataset to do my research.

**Comment by Th3lios** [comment]
MingkeVan, did you get the DEAP dataset? I'm writing a thesis and I would like to know if you got the dataset and if you can share it.

**Comment by Arka95** [comment]
Yes, they ususally reply in 14 to 20 days. I got a late reply from them.
They will send you the password and account details . You have to mail them
from a university registered email id.

**Comment by adityajapa** [comment]
how can i get SEED dataset

**Comment by dtzfast** [comment]
I want to know if this project or the code has corresponding articles. Thank you very much.
I have never found a code and article corresponding to it. I especially need this kind of information to learn. thank you very much

**Comment by GeorgeKBarsoum** [comment]
I want to access DEAP Data set I  sent the ELUA about two weeks ago
and I didn't get any reply yet
Can anyone share it with me?

**Comment by Arka95** [comment]
yeah. they take time . it took me over a month to get a reply. Try
resending. Their EULA prohobits anyone from directly sharing the data.
